clouds/databricks/core/src/main/resources/sql/createUDFs.sql

```diff
@@ -0,0 +1,90 @@
+CREATE OR REPLACE FUNCTION st_antimeridianSafeGeom as 'com.carto.analyticstoolbox.core.ST_AntimeridianSafeGeom';
```

Comment:
These are the functions that we called core right? Is it something that we will might be modifying in the future adding more? or they are provided by a third party? In the first case maybe we can generate this sql automatically be readying the other sqls. I wouldn't consider it a priority though.

Reply:
The package is called core at the moment. For now, they have not been migrated yet, so I just left the sql, but not the UDF that they are calling. 
When we migrate the modules they will have a package like this:
`CREATE OR REPLACE FUNCTION dummy_function as 'com.carto.analyticstoolbox.modules.dummy.DummyFunction';`
The Idea is having each sql script within its own module, although considering that a full package is always going to be delivered, it could be also a good idea to have it centralized in one global script (as is now)